.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
```

Comment:
Nice way to go from 46% coverage to 94%, don't test all the env files!  Not to mention our coverage.py runs way faster when its not looking through like, 3000 files.